sync/src/synchronizer/block_fetcher.rs

```diff
@@ -1,5 +1,5 @@
-use super::header_view::HeaderView;
 use crate::synchronizer::{BlockStatus, Synchronizer};
+use crate::types::HeaderView;
```

Comment:
I don't see that the mod `header_view` is removed, so now we have two different `HeaderVIew`?

Reply:
ops, I removed mod ref but forgot to delete this file. will push it later.